### ISSUE TITLE
ImportError: No module named 'KECA'

### ISSUE THREAD

**Issue by BlackArbsCEO** [tsterbak/kernel_eca#1]
Hi,

I tried to run your example and had an importerror. See below:

![kernel_eca missing module](https://cloud.githubusercontent.com/assets/7452471/12997284/f9d0b3d4-d0f4-11e5-943e-4fe1425d0013.png)

**Comment by tsterbak** [comment]
Yeah I know... i don't know how to resolve this issue, if you know sth feel free.

If you install the module this code should run without problems

**Comment by indrajitsg** [comment]
You just need to use:

````
from kernel_eca import KernelECA
````
